doc/user_guide/user_guide.md

```diff
@@ -218,12 +218,22 @@ The same style of overloaded function exists for text content upload too in the
 
 Unless you really need it and know exactly what you are doing, we recommend to stick to blocking operation for your tests.
 
+### Delete a File from BucketFS
+
+Deleting a file is straight forward:
+
+```java
+bucket.deleteFileNonBlocking(fileName);
+```
```

Comment:
Should we change this to `deleteFile` and accept another boolean `isBlocking` parameter? By default it is set for non-blocking?

Is there plan to support blocking version also?

Reply:
I'm unsure about that. I picked that name because the write methods are named like that and have an additional blocking method. 
I don't really know if BucketFs waits until the files are deleted from all nodes. It's also hard to tests since behavior could be different for single / multi node or other parameters.
@redcatbear what do you think?

Reply:
The only way to be sure is again to monitor the log file if you want blocking operation. My money is on this request returning without waiting for the file to be deleted.

Reply:
ok then let's keep the name as it is, and if it shows that we need a method with checking, we add `deleteFileBlocking`